src/components/layout.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { UnifiedFooter } from '@mdb/consistent-nav';
+import { Global, css } from '@emotion/react';
+
+import ConsistentNav from './consistent-nav';
+import { colorMap, size, fontSize, screenSize, lineHeight } from './theme';
+
+const FOOTER_LOGO_WIDTH = '152px';
+
+const globalStyles = css`
+    html {
+        box-sizing: border-box;
+        height: 100%;
+    }
+    *,
+    *::before,
+    *::after {
+        box-sizing: inherit;
+    }
+    body {
+        background: ${colorMap.pageBackground};
+        color: ${colorMap.devWhite};
+        font-family: akzidenz, -apple-system, BlinkMacSystemFont, 'Segoe UI',
+            Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+            'Segoe UI Symbol';
+        font-size: ${fontSize.default};
+        line-height: ${lineHeight.medium};
+        margin: 0;
+        padding: 0;
+    }
```

Comment:
Not sure if it's emotion format, but can we have the styles in separate file ? I might be dumb here, but I always felt with this format , the main component and it's functionality are being lost with a lot of styling code above it.

Reply:
Yeah I agree with you, it's a bit distracting when most of the file is styling. This is the way I've seen it in University/DevHub, but just did some research and there is also a convention of separating styled components into their own directory (for this is would be `src/components/layout.tsx` and `src/styled/layout.tsx`). Adds more files, but might make this clearer?

Reply:
Correct. For me personally I prefer a separate file. @gfunk1230 any thoughts on this ?

Reply:
Yep! I would do that but I thought NextJS already had a built-in global file for CSS like `styles/globals.css` but if not I would create a global styles file and import that. Also, we should add CSS resets in that same file. Here is a good article on resets https://css-tricks.com/an-interview-with-elad-shechter-on-the-new-css-reset/

Reply:
Yeah I think Next gets initialized with that global CSS file, but I just took it out since we were using emotion, and we rely on the theme for a lot of these styles, so it's nice to do the CSS in JS thing. I could just create a styled component in `src/styled/global.tsx` and then import that? And cool cool I'll look into resets!